src/components/FileUploader/FileUploader-test.js

```diff
@@ -55,6 +55,15 @@ describe('FileUploaderButton', () => {
     it('does not have default role', () => {
       expect(mountWrapper.props().role).not.toBeTruthy();
     });
+
+    it('resets the input value onClick', () => {
+      const input = mountWrapper.find('.bx--visually-hidden');
+      input.instance().value = '';
+      const evt = { target: { value: input.instance().value } };
+      input.simulate('click', evt);
+
```

Comment:
This feels so funky, but since the onClick function just clears the elements value I couldn't figure out a good way to pass the element's value in and get it out without wrapping it in the mock evt object.